### PR TITLE
Katas for week 1 & 2

### PR DESCRIPTION
**Pull request by Luisjor**

### CODE REVIEW

File: katas/jorge-chavez/01-30/Greed Is Good/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Greed is a dice game played with five six-sided dice. Your mission, should you choose to accept it, is to score a throw according to these rules. You will always be given an array with five six-sided dice values.
```
Comment (sebasonti-jalauni) [inline katas/jorge-chavez/01-30/Greed Is Good/README.md:1]: Kata link is missing here

Comment (Luisjor) [reply katas/jorge-chavez/01-30/Greed Is Good/README.md:3]: Thank you, I already added it.

File: katas/jorge-chavez/02-06/ROT 13/ROT13.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+namespace katas.JorgeChavez;
+
+public class ROT13
+{
+    public static string Rot13(string message)
+    {
+        var alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
```
Comment (sebasonti-jalauni) [inline katas/jorge-chavez/02-06/ROT 13/ROT13.cs:7]: Good implementation of the Ceasar cypher, however, how would you implement a more general solution for a Caesar cypher that shifts the letters of a message to any value, for example 7 letters, or 19 letters?

File: katas/jorge-chavez/02-13/Sudoku Validator/SudokuValidator.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
+public class SudokuValidator
```
Comment (sebasonti-jalauni) [inline katas/jorge-chavez/02-13/Sudoku Validator/SudokuValidator.cs:3]: Please remove this kata from this PR, I won't be able to review it just yet and we need to merge this PR

**Review by sebasonti-jalauni — CHANGES_REQUESTED** [review]

File: katas/jorge-chavez/01-30/Sum Of Digits/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Digital root is the recursive sum of all the digits in a number.
```
[the remark this answers is not in the document]
Comment (Luisjor) [reply katas/jorge-chavez/01-30/Sum Of Digits/README.md]: Done

File: katas/jorge-chavez/01-30/Valid Parentheses/README.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+https://www.codewars.com/kata/52774a314c2333f0a7000688s
```
[the remark this answers is not in the document]
Comment (Luisjor) [reply katas/jorge-chavez/01-30/Valid Parentheses/README.md]: Solved

File: katas/jorge-chavez/02-13/Sudoku Validator/SudokuValidator.cs (hunk 3 above)
Comment (Luisjor) [reply katas/jorge-chavez/02-13/Sudoku Validator/SudokuValidator.cs]: Done

File: katas/jorge-chavez/01-30/Sum Of Digits/SumOfDigits.cs [hunk 6]
```diff
@@ -0,0 +1,23 @@
+namespace katas.JorgeChavez;
+
+public class Number
+{
+    public static int DigitalRoot(long n)
+    {
+        string numText = n.ToString();
```
[the remark this answers is not in the document]
Comment (Luisjor) [reply katas/jorge-chavez/01-30/Sum Of Digits/SumOfDigits.cs:7]: Noted, thank you

**Review by sebasonti-jalauni — APPROVED** [review]

**Review by svaleria — APPROVED** [review]

**Pull request merged**